js/bundle_test.go

```diff
@@ -729,32 +768,51 @@ func TestOpen(t *testing.T) {
 }
 
 func TestBundleInstantiate(t *testing.T) {
-	b, err := getSimpleBundle(t, "/script.js", `
+	t.Parallel()
+	t.Run("Run", func(t *testing.T) {
+		t.Parallel()
+		b, err := getSimpleBundle(t, "/script.js", `
 		export let options = {
 			vus: 5,
 			teardownTimeout: '1s',
 		};
 		let val = true;
 		export default function() { return val; }
 	`)
-	if !assert.NoError(t, err) {
-		return
-	}
-	logger := testutils.NewLogger(t)
-
-	bi, err := b.Instantiate(logger, 0)
-	if !assert.NoError(t, err) {
-		return
-	}
+		if !assert.NoError(t, err) {
+			return
+		}
```

Comment:
This pattern also repeats a lot below
```suggestion
		require.NoError(t, err)
```